lib/ProfileData/SampleProfWriter.cpp

```diff
@@ -122,5 +122,5 @@ SampleProfileWriter::create(StringRef Filename, SampleProfileFormat Format) {
   if (EC)
     return EC;
 
-  return std::move(Writer);
```

Comment:
This likely cuts both ways. Clang and GCC disagree about when std::move is valid, and that behavior has changed based on which C++ language version you're targeting.

Reply:
:+1:  FWIW, clang 16.0.3 appears to be "happy" with and without `std::move`.